api/fluentbitoperator/v1alpha2/plugins/input/tail_types.go

```diff
@@ -87,6 +87,9 @@ type Tail struct {
 	DockerModeFlushSeconds *int64 `json:"dockerModeFlushSeconds,omitempty"`
 	// DisableInotifyWatcher will disable inotify and use the file stat watcher instead.
 	DisableInotifyWatcher *bool `json:"disableInotifyWatcher,omitempty"`
+	// This will help to reassembly multiline messages originally split by Docker or CRI
+	//Specify one or Multiline Parser definition to apply to the content.
+	MultilineParser string `json:"multilineParser,omitempty"`
```

Comment:
It is not an appropriate practice to define `MultilineParser` as a string.

What if the parameter `multiline.xxx` is added later?

Maybe a struct is more reasonable.

```
type Multiline struct {
  Parser string
}
```

Reply:
Thanks. But there is the `multiline` property in the old property.
```
	// If enabled, the plugin will try to discover multiline messages
	// and use the proper parsers to compose the outgoing messages.
	// Note that when this option is enabled the Parser option is not used.
	Multiline *bool `json:"multiline,omitempty"`
```
And after testing, and if a structure is added, the following input is required for the user
```
DB:
      sync: xx
```
Not like the original
```
dbSync: xxx
````
The original configuration file is more similar to fluent-bit itself